prow/cmd/branchprotector/dev-job.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: branchprotector
+spec:
+  backoffLimit: 2
+  template:
+    metadata:
+      labels:
+        name: branchprotector
+    spec:
+      containers:
+      - name: branchprotector
+        image: gcr.io/k8s-testimages/branchprotector:latest  # Note: not gcr.io/k8s-prow for dev
```

Comment:
I think you want `image_chroot`, and you can use the same template, let's chat :)